app/views/events/_show_event_large.html.erb

```diff
@@ -7,7 +7,13 @@
   </div>
   <h1><%= @event.name %></h1>
 </div>
-<%= image_tag "event-detail.png", class: "img-rounded img-responsive center-block m-t-1 m-b-1" %>
+<div id="wsp-event-show-banner"></div>
+<script>
```

Comment:
Ah short comment about the purpose of the script would be good. Something like: "Generate random image, in order to not have the same static image everywhere". So later no one will wonder why this script is necessary.